client/post-editor/editor-diff-viewer/index.jsx

```diff
@@ -99,7 +114,10 @@ class EditorDiffViewer extends PureComponent {
 		);
 	};
 
-	debouncedRecomputeChanges = debounce( partial( this.recomputeChanges, null ), 500 );
+	debouncedRecomputeChanges = debounce( () => {
+		this.isBigViewport = isWithinBreakpoint( '>1040px' );
+		partial( this.recomputeChanges, null )();
```

Comment:
why partial here? `this.recomputeChanges( null )` should work too?

Reply:
You're right! I'll correct that, thanks.